src/cart/crt_group.c

```diff
@@ -2566,6 +2566,7 @@ crt_rank_self_set(d_rank_t rank)
 
 	D_RWLOCK_WRLOCK(&default_grp_priv->gp_rwlock);
 	default_grp_priv->gp_self = rank;
+	default_grp_priv->gp_membs_ver_min = group_version_min;
```

Comment:
You use value of 0 as a special value for 'no version set yet', so we probably should check incoming group_version_min for that and error out if its 0

Reply:
Indeed. An earlier revision of this patch actually had a control issue that would be easier to debug with such a check. Thanks, Alex. Let me add it...